@here/harp-mapview/lib/copyrights/CopyrightInfo.ts

```diff
@@ -110,7 +112,9 @@ export namespace CopyrightInfo {
                 .map(entry => {
                     const label = entry.label !== undefined ? entry.label : entry.id;
                     const text = entry.year !== undefined ? `${entry.year} ${label}` : label;
-                    return entry.link ? `<a href="${entry.link}">${text}</a>` : `${text}`;
+                    return entry.link
+                        ? `<a href="${entry.link}" target="_blank">${text}</a>`
```

Comment:
Please also add `rel="noreferrer"` and `rel="noopener"`
According to: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target
Using target="_blank" without rel="noreferrer" and rel="noopener" makes the website vulnerable to window.opener API exploitation attacks (vulnerability description).

Reply:
Fixed, thank you for pointing out to this.